src/ProjectService.Mappers/ModelsMappers/TaskInfoMapper.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.ProjectService.Mappers.ModelsMappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.ModelsMappers
+{
+    public class TaskInfoMapper : ITaskInfoMapper
+    {
+        public TaskInfo Map(DbTask dbTask, IGetUserDataResponse assignedUser, IGetUserDataResponse author)
+        {
+            if (dbTask == null)
+            {
+                throw new ArgumentNullException(nameof(dbTask));
+            }
+
+            var db = new TaskInfo
```

Comment:
use return here

Reply:
why you called 'db'? may be 'taskInfo'

Reply:
no i did it for the test but forgot to delete

Reply:
Fixed